doc/source/whatsnew/v1.1.1.rst

```diff
@@ -44,6 +44,10 @@ Bug fixes
 
 -
 
+**Groupby/resample/rolling**
+
+- Bug in :class:`pandas.core.groupby.RollingGroupby` where ``__getitem__`` would not select the specified column (:issue:`35486`)
```

Comment:
maybe move to regression section above.

(this section is for bug fixes, we maybe should have a discussion in #35489 regarding the criteria for backporting bugfixes otherwise this section may end up empty)